test/helpers/make-context.js

```diff
@@ -0,0 +1,22 @@
+var BufferList = require('bl');
+
+function makeContext(mod, path, options) {
```

Comment:
So if a test needs a specialised context, they need to write their own version of `makeContext` instead of just using this one? If so then I'd say a comment at the top of this file explaining its limitations would be fine. You only need to know about its limitations if you're planning to use it right? I guess you could call it `makeGenericContext` or something, but seems a bit overkill.